metricshub-doc/src/site/markdown/usecases/network-interfaces.md

```diff
@@ -0,0 +1,61 @@
+keywords: snmp
+description: How to configure MetricsHub to monitor network interfaces using the SNMP protocol.
+
+# Monitoring network interfaces using SNMP
```

Comment:
The use case really covers Lenovo servers, and not network cards through SNMP v2. Instead of a Lenovo server, we should rather use an exotic network device, where only MIB-2 is available.